bucket/janet.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.13.1",
+    "description": "Janet is a lisp-like, embeddable, functional and imperative programming language and bytecode interpreter.",
+    "homepage": "https://janet-lang.org",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x64-installer.msi",
+            "hash": "ef5d32257594c8ab54aaa52fe8e84bb951a4883bbb6f6848e845759122d33a35"
+        },
+        "32bit": {
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x86-installer.msi",
+            "hash": "627a0dccef7815fdde735caf3ea2eb3e16f2ec26ac3d0cc365cf8ea67589d5e3"
+        }
+    },
+    "extract_dir": "Janet",
+    "bin": [
+        "bin\\janet.exe",
+        "bin\\jpm.bat"
+    ],
+    "env_set": {
+        "JANET_BINPATH": "$dir\\bin",
+        "JANET_PATH": "$dir\\Library",
+        "JANET_HEADERPATH": "$dir\\C",
+        "JANET_LIBPATH": "$dir\\C"
```

Comment:
Documentation? Anything, why all of these are needed?

Reply:
I believe that just to run `janet` they're actually not needed. You need them once you want to link / embed with `libjanet`. I'll try to figure this out. Maybe they can actually be removed as well.

Reply:
Based on [jpm man page](https://github.com/janet-lang/janet/blob/master/jpm.1#L212-L266) I ended up with following conclusions:
- it seem that only `JANET_PATH` is needed by so called `normal installation`
- `JANET_PATH` should also be `persistent` between versions

I'll update manifest accordingly.

Reply:
I extracted the findings to description.